<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>398</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>База данных</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout_3">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Теущая таблица:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbChooseTable"/>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QTableView" name="tvModel">
      <property name="minimumSize">
       <size>
        <width>510</width>
        <height>250</height>
       </size>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Выборка</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Столбец</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ComboBoxForDB" name="cbSelectColumn"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Значение</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QTextEdit" name="teDecript"/>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="Search">
           <property name="text">
            <string>Найти</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="RevertTable">
           <property name="text">
            <string>Восстановить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="0">
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="SaveButton">
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="RevertButton">
        <property name="text">
         <string>Отменить</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="Add">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="Delete">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="Reconnect">
      <property name="text">
       <string>Переподключение</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBoxForDB</class>
   <extends>QComboBox</extends>
   <header>comboboxfordb.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
